src/app/pages/questions/components/question/question.component.ts

```diff
@@ -210,4 +215,9 @@ export class QuestionComponent implements OnInit, OnChanges {
       behavior: 'smooth'
     })
   }
+
+  onAudioRecordStart(start: boolean) {
+    if (start) this.nextAction.emit(NextButtonEventType.DISABLE)
+    else this.nextAction.emit(NextButtonEventType.ENABLE)
```

Comment:
Might also be useful to have a pop-up dialog saying "Please stop the audio recording once you are done to enable the next button" when they press the disabled Next button.

Reply:
Yes I can add this.